examples/datafeed.py

```diff
@@ -3,27 +3,20 @@
 import logging.config
 import os
 
+from symphony.bdk.core.activity.command import CommandContext
 from symphony.bdk.core.config.loader import BdkConfigLoader
-from symphony.bdk.core.service.datafeed.real_time_event_listener import RealTimeEventListener
 from symphony.bdk.core.symphony_bdk import SymphonyBdk
-from symphony.bdk.gen.agent_model.v4_initiator import V4Initiator
-from symphony.bdk.gen.agent_model.v4_message_sent import V4MessageSent
 
 
 async def run():
     config = BdkConfigLoader.load_from_symphony_dir("config.yaml")
 
     async with SymphonyBdk(config) as bdk:
-        datafeed_loop = bdk.datafeed()
-        datafeed_loop.subscribe(RealTimeEventListenerImpl())
-        await datafeed_loop.start()
+        @bdk.slash("/hello")
```

Comment:
can we use it out of the with block?

Reply:
probably not it is using the bdk instance right?

same as

```
from flask import Flask
app = Flask(__name__)

@app.route('/')
def hello_world():
    return 'Hello, World!'
```

Reply:
Yes it is using the bdk instance. But a user could not use a `with` block and manually call the `close_clients` method in a `finally` block.